bool open
int32 armId
---

